test/unit/mux.spec.mjs

```diff
@@ -0,0 +1,15 @@
+import { expect } from 'chai';
+import Mux from '../../esm/index.mjs';
+
+/** @test {Mux} */
+describe('ESM: Unit::Mux', () => {
+  /** @test {Mux} */
+  describe('Mux', () => {
+    /** @test {Mux} */
+    it('exposes Mux via ESM default', () => {
+      const muxClient = new Mux('testKey', 'testSecret');
+
+      expect(muxClient).to.to.be.an.instanceof(Mux);
```

Comment:
Did you really need `to.to.be` here instead of just `to.be`?

Reply:
Fixed